helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py

```diff
@@ -39,9 +39,24 @@ def _prepare_vals(self):
 
     def action_helpdesk_ticket_to_lead(self):
         lead = self.env["crm.lead"].create(self._prepare_vals())
+        for follower in self.ticket_id.message_follower_ids:
+            lead.message_subscribe(
+                partner_ids=[follower.partner_id.id],
+                subtype_ids=follower.subtype_ids.ids,
+            )
         self.ticket_id.write({"lead_ids": [(4, lead.id)]})
         for message in self.ticket_id.message_ids:
-            message.copy({"model": lead._name, "res_id": lead.id})
+            new_message = message.copy(
+                {
+                    "model": lead._name,
+                    "res_id": lead.id,
+                    # prevent null value in column "notification_type" if message
+                    # have notifications (not copied)
+                    "notified_partner_ids": False,
+                }
+            )
+            for notification in message.notification_ids:
```

Comment:
Why do you need to do this?

Reply:
Do you think it is not necessary to transfer notifications?

Reply:
I don't think so.